docs/docs/clients/server-side.md

```diff
@@ -1156,36 +1156,40 @@ _flagsmithClient = new FlagsmithClient(
     # Override the default Flagsmith API URL if you are self-hosting.
     # Optional.
     # Defaults to https://edge.api.flagsmith.com/api/v1/
-    apiUrl: "https://flagsmith.myproject.com"
+    apiUrl: "https://flagsmith.myproject.com",
+
+    # Enables caching support.
```

Comment:
This should be lower down in the config options - follow the other languages that have caching